### DISCUSSION TITLE
Zettels in root folder

### DISCUSSION THREAD

**Discussion by metamoof** [bramses/bramses-highly-opinionated-vault-2023#3] [Q&A]
So, I'm new to Zettlekasen and BHOV, and I've only been using obsidian a little while. You've done impressive work with the README, but it's a little overwhelming at first.

I'm trying to work out the reasoning for keeping zettels in the root folder, rather than in a subfolder, or in a daily folder. It looks like it has a potential to "clutter" an otherwise sensibly ordered set of notes.

 I could understand it if the zettels were supposed tobe ephemera, to be deleted at a later date (something I'm unsure would work for me), but that doesn't seem to be part of the philosophy.

So, why in the root?

**Comment by bramses** [comment]
Great question! 

The short answer: Zettels have their own categorization with tags (the Zettelkasten template automatically adds the #zettel tag)

In the "Computed" folder you can virtualize a folder by leveraging dataview.

The Luhman plug-in allows for searching/inserting any Zettel that follows the ID protocol of "unique ID"

All that said, you can switch the Unique Note creator to add notes to a sub folder of your choice.

Hope this cleared a bit up!

**Reply by metamoof** [reply]
I think I'd prefer to do that, if nothing else because I run on Windows and from experience the file system tends to get a bit slow and janky when you end up with a few thousand items in one folder. As such, I aggressively optimise file storage into saving in a per month or per year folder when there seems to be no sensible horizon for future cap on files and no other more sensible approach seems likely. It's not an issue *now*, but it will be a few years down the line, and we need to take that into account.

I'll have a play with it some more, see if it makes more sense to me with that modification.
